adbe/adb_enhanced.py

```diff
@@ -50,6 +50,9 @@
 # Value to be return as 'unknown' to the user
 _USER_PRINT_VALUE_UNKNOWN = 'unknown'
 
+SCREEN_ON = 1
```

Comment:
Nit. Consider prefixing them with "_" to make them private.

Reply:
But it's used in `main.py`
Should i extract it into another file maybe?

Reply:
My bad. Leave it as-is then.